gcloud-java-speech/src/main/java/com/google/cloud/speech/spi/v1beta1/SpeechApi.java

```diff
@@ -0,0 +1,311 @@
+import com.google.cloud.speech.v1beta1.AsyncRecognizeRequest;
+import com.google.cloud.speech.v1beta1.RecognitionAudio;
+import com.google.cloud.speech.v1beta1.RecognitionConfig;
+import com.google.cloud.speech.v1beta1.SyncRecognizeRequest;
+import com.google.cloud.speech.v1beta1.SyncRecognizeResponse;
+import com.google.longrunning.Operation;
+import io.grpc.ManagedChannel;
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ScheduledExecutorService;
+
+// AUTO-GENERATED DOCUMENTATION AND SERVICE
+/**
+ * Service Description: Service that implements Google Cloud Speech API.
+ *
+ * <p>This class provides the ability to make remote calls to the backing service through method
+ * calls that map to API methods. Sample code to get started:
+ *
+ * <pre>
+ * <code>
+ * try (SpeechApi speechApi = SpeechApi.create()) {
+ *   RecognitionConfig config = RecognitionConfig.newBuilder().build();
+ *   RecognitionAudio audio = RecognitionAudio.newBuilder().build();
+ *   SyncRecognizeResponse response = speechApi.syncRecognize(config, audio);
+ * }
+ * </code>
+ * </pre>
+ *
+ * <p>Note: close() needs to be called on the speechApi object to clean up resources such
+ * as threads. In the example above, try-with-resources is used, which automatically calls
+ * close().
+ *
+ * <p>The surface of this class includes several types of Java methods for each of the API's methods:
+ *
+ * <ol>
+ * <li> A "flattened" method. With this type of method, the fields of the request type have been
+ * converted into function parameters. It may be the case that not all fields are available
+ * as parameters, and not every API method will have a flattened method entry point.
+ * <li> A "request object" method. This type of method only takes one parameter, a request
+ * object, which must be constructed before the call. Not every API method will have a request
+ * object method.
+ * <li> A "callable" method. This type of method takes no parameters and returns an immutable
+ * ApiCallable object, which can be used to initiate calls to the service.
+ * </ol>
+ *
+ * <p>See the individual methods for example code.
+ *
+ * <p>Many parameters require resource names to be formatted in a particular way. To assist
+ * with these names, this class includes a format method for each type of name, and additionally
+ * a parse method to extract the individual identifiers contained within names that are
+ * returned.
+ *
+ * <p>This class can be customized by passing in a custom instance of SpeechSettings to
+ * create(). For example:
+ *
+ * <pre>
+ * <code>
+ * SpeechSettings speechSettings = SpeechSettings.defaultBuilder()
+ *     .provideChannelWith(myCredentials)
+ *     .build();
+ * SpeechApi speechApi = SpeechApi.create(speechSettings);
+ * </code>
+ * </pre>
+ */
+@javax.annotation.Generated("by GAPIC")
+public class SpeechApi implements AutoCloseable {
```

Comment:
We specifically excluded it because we don't support it yet; the surface of streaming methods needs to be different. We are working on that separately.